Front-end/glaucoma-diagnosis-system/src/App.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-<div>
+<div class="bg-cover min-h-screen" style="background-image: url('https://images.unsplash.com/photo-1519681393784-d120267933ba?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1124&q=100');'">
```

Comment:
I don't like the background image. I think the previous one with no bg image is better, wdyt?